content/docs/user-guide/project-structure/index.md

```diff
@@ -5,9 +5,10 @@ project</abbr>, including the internal `.dvc/` directory. From there on, you
 will create and manage different DVC metafiles (below), and populate the
 <abbr>cache</abbr> with data artifacts as you work on your ML experiments.
 
-- `dvc.yaml` files define stages that form the pipeline(s) of a project. All
-  stage-based features such as `dvc params`, `dvc metrics`, and `dvc plots` are
-  specified here.
+- `dvc.yaml` files define stages that form the pipeline(s) of a project, and
```

Comment:
this feel a bit indirect (feels to me that we talk about a specific eval stage)

Reply:
Reworded. PTAL.